### ISSUE TITLE
Tag releases on Docker Hub

### ISSUE THREAD

**Issue by mbarrien** [pomerium/pomerium#55]
Now that there are getting to be releases of Pomerium, there's build artifacts in the form of releases on GitHub, but it would be nice to have a tag for a Docker Hub image corresponding to that same release. As much as I believe in CI/CD, sometimes we don't want to always be on the latest master branch image.

**Comment by desimone** [comment]
Agreed. I just pushed v0.0.2 (`pomerium/pomerium:v0.0.2`) to dockerhub. 

I set up automatic tag on docker-hub but it looks like it didn't fire. I can't recall if I didn't it after the fact, so that may be why. Hopefully it will work on next release cut. 

<img width="916" alt="screen shot 2019-02-19 at 8 23 40 pm" src="https://user-images.githubusercontent.com/1544881/53066437-bba52300-3484-11e9-8c31-c32cc3717d3c.png">

<img width="916" alt="screen shot 2019-02-19 at 8 23 40 pm" src="https://user-images.githubusercontent.com/1544881/53066359-505b5100-3484-11e9-8715-f8fec5e7b8bd.png">

See:
https://stackoverflow.com/questions/53802775/automated-builds-wit-github-tags
